app/assets/javascripts/components/app.js.jsx

```diff
@@ -2,9 +2,16 @@ class App extends React.Component {
   render () {
     return (
       <div>
-        <ul>
+        <section id="header">
+          <img alt="icon" src="/assets/top.png"/>
+          <h1>EatWell.<br/>Let's find lunch<br/>now...</h1>
```

Comment:
Usually, makes sense to use headers of various levels (h1,h2,h3) instead of br in header.